diff --git a/test/gemfiles/rails_6_0.gemfile b/test/gemfiles/rails_6_0.gemfile
index daeffa5..44e38be 100644
--- a/test/gemfiles/rails_6_0.gemfile
+++ b/test/gemfiles/rails_6_0.gemfile
@@ -5,3 +5,9 @@ gem 'activesupport', '~> 6.0.3'
 gem 'sassc-rails', '~> 2.0'
 
 gemspec path: '../../'
+
+gem 'base64'
+gem 'drb'
+gem 'mutex_m'
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../../..'
diff --git a/test/gemfiles/rails_6_1.gemfile b/test/gemfiles/rails_6_1.gemfile
index 04342fb..e59f402 100644
--- a/test/gemfiles/rails_6_1.gemfile
+++ b/test/gemfiles/rails_6_1.gemfile
@@ -5,3 +5,9 @@ gem 'activesupport', '~> 6.1.3'
 gem 'sassc-rails', '~> 2.0'
 
 gemspec path: '../../'
+
+gem 'base64'
+gem 'drb'
+gem 'mutex_m'
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../../..'
diff --git a/test/gemfiles/rails_7_0_dartsass.gemfile b/test/gemfiles/rails_7_0_dartsass.gemfile
index 6df9142..759ab8d 100644
--- a/test/gemfiles/rails_7_0_dartsass.gemfile
+++ b/test/gemfiles/rails_7_0_dartsass.gemfile
@@ -5,3 +5,8 @@ gem 'activesupport', '~> 7.0.4'
 gem 'dartsass-sprockets', '~> 3.0'
 
 gemspec path: '../../'
+
+gem 'base64'
+gem 'drb'
+gem 'mutex_m'
+gem 'sassc-embedded', path: '../../../../../..'
diff --git a/test/gemfiles/rails_7_0_sassc.gemfile b/test/gemfiles/rails_7_0_sassc.gemfile
index 58fc039..aa8c54b 100644
--- a/test/gemfiles/rails_7_0_sassc.gemfile
+++ b/test/gemfiles/rails_7_0_sassc.gemfile
@@ -5,3 +5,9 @@ gem 'activesupport', '~> 7.0.4'
 gem 'sassc-rails', '~> 2.0'
 
 gemspec path: '../../'
+
+gem 'base64'
+gem 'drb'
+gem 'mutex_m'
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../../..'
diff --git a/test/test_helper.rb b/test/test_helper.rb
index da89690..dd535dc 100644
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -34,9 +34,9 @@ end
 
 Capybara.configure do |config|
   config.server = :webrick
-  config.app_host = 'http://localhost:7000'
+  config.app_host = 'http://localhost:8000'
   config.default_driver = :cuprite
   config.javascript_driver = :cuprite
-  config.server_port = 7000
+  config.server_port = 8000
   config.default_max_wait_time = 10
 end
