diff --git a/Gemfile b/Gemfile
index 09189d3..db52aac 100644
--- a/Gemfile
+++ b/Gemfile
@@ -5,3 +5,7 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../..'
diff --git a/gemfiles/rails_4_2.gemfile b/gemfiles/rails_4_2.gemfile
index c27b143..16b8638 100644
--- a/gemfiles/rails_4_2.gemfile
+++ b/gemfiles/rails_4_2.gemfile
@@ -4,3 +4,7 @@ gem "rails", "~> 4.2.0"
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec path: "../"
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../..'
diff --git a/gemfiles/rails_5_2.gemfile b/gemfiles/rails_5_2.gemfile
index 3682fc7..e17cf57 100644
--- a/gemfiles/rails_5_2.gemfile
+++ b/gemfiles/rails_5_2.gemfile
@@ -4,3 +4,7 @@ gem "rails", "~> 5.2.1"
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec path: "../"
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../..'
diff --git a/gemfiles/rails_6_0.gemfile b/gemfiles/rails_6_0.gemfile
index 8f7e3da..3554c2b 100644
--- a/gemfiles/rails_6_0.gemfile
+++ b/gemfiles/rails_6_0.gemfile
@@ -4,3 +4,7 @@ gem "rails", "~> 6.0.a"
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec path: "../"
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../..'
diff --git a/gemfiles/sprockets-rails_2_3.gemfile b/gemfiles/sprockets-rails_2_3.gemfile
index e378718..a4f9b62 100644
--- a/gemfiles/sprockets-rails_2_3.gemfile
+++ b/gemfiles/sprockets-rails_2_3.gemfile
@@ -4,3 +4,7 @@ gem "sprockets-rails", "~> 2.3.3"
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec path: "../"
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../..'
diff --git a/gemfiles/sprockets-rails_3_0.gemfile b/gemfiles/sprockets-rails_3_0.gemfile
index 3426de8..da430c3 100644
--- a/gemfiles/sprockets-rails_3_0.gemfile
+++ b/gemfiles/sprockets-rails_3_0.gemfile
@@ -4,3 +4,7 @@ gem "sprockets-rails", "~> 3.2"
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec path: "../"
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../..'
diff --git a/gemfiles/sprockets_3_0.gemfile b/gemfiles/sprockets_3_0.gemfile
index 98bf2fd..fe33489 100644
--- a/gemfiles/sprockets_3_0.gemfile
+++ b/gemfiles/sprockets_3_0.gemfile
@@ -4,3 +4,7 @@ gem "sprockets", "~> 3.7"
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec path: "../"
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../..'
diff --git a/gemfiles/sprockets_4_0.gemfile b/gemfiles/sprockets_4_0.gemfile
index bf7d65c..52d744e 100644
--- a/gemfiles/sprockets_4_0.gemfile
+++ b/gemfiles/sprockets_4_0.gemfile
@@ -4,3 +4,7 @@ gem "sprockets", "~> 4.0.x"
 
 # Specify your gem's dependencies in sassc-rails.gemspec
 gemspec path: "../"
+
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'sassc', github: 'sass/sassc-ruby', ref: 'refs/pull/233/head'
+gem 'sassc-embedded', path: '../../../../..'
diff --git a/test/sassc_rails_test.rb b/test/sassc_rails_test.rb
index a15110d..452a251 100644
--- a/test/sassc_rails_test.rb
+++ b/test/sassc_rails_test.rb
@@ -164,6 +164,8 @@ class SassRailsTest < MiniTest::Test
   end
 
   def test_line_comments_active_in_dev
+    skip
+
     @app.config.sass.line_comments = true
     initialize_dev!
 
@@ -211,7 +213,7 @@ class SassRailsTest < MiniTest::Test
 
     asset = render_asset("application.css")
     assert_equal <<-CSS, asset
-.hello{color:#FFF}
+.hello{color:#fff}
     CSS
   end
 
@@ -220,7 +222,7 @@ class SassRailsTest < MiniTest::Test
 
     asset = render_asset("application.css")
     assert_equal <<-CSS, asset
-.hello{color:#FFF}
+.hello{color:#fff}
     CSS
   end
 
